research/src/pages/prototypes/selectmenu.mdx

```diff
@@ -156,18 +156,18 @@ You can replace the default listbox part in a similar fashion:
 </style>
 <selectmenu class="my-custom-select">
   <div slot="listbox">
-    <popup behavior="listbox">
+    <div popup="popup" behavior="listbox">
       <option>Option 1</option>
       <option>Option 2</option>
       <option>Option 3</option>
       <option>Option 4</option>
       <option>Option 5</option>
-    </popup>
+    </div>
   </div>
 </selectmenu>
 ```
 
-The `<popup>` used here is the element proposed at [Popup (Editor's Draft)](http://localhost:8000/components/popup). The element with `behavior="listbox"` is required to be a `<popup>`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
+The `<div popup>` used here is the element proposed at [Popup (Editor's Draft)](http://localhost:8000/components/popup). The element with `behavior="listbox"` is required to be a `<div popup>`. Applying `behavior="listbox"` tells the browser to apply select menu listbox behavior to that element: it will be opened when the `<selectmenu>`'s button is clicked, and the user can select `<option>`s inside it with mouse, arrow keys, and touch.
```

Comment:
should this be `<div popup=popup>` to match the examples used in the popup explainer?

<img width="404" alt="image" src="https://user-images.githubusercontent.com/17087167/163999249-48b80ac0-1a1b-420b-a525-2e70a4a29e82.png">

Reply:
Thank you for catching this! I had missed that spot.